Day-II-Calculator/calculator/src/components/ButtonComponents/ActionButton.js

```diff
@@ -1,2 +1,10 @@
 import React from 'react';
-import './Button.css';
+import './Button.sass';
+
+const ActionButton = (props) => {
+
+  return <button className="action" onClick={() => props.onClick(props.action)}>{props.action}</button>;
```

Comment:
You could use [object destructuring](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#Object_destructuring) for this:
```javascript
const ActionButton = ({ action, onClick }) => { 
  return (
    <button
      className="action"
      onClick={() => onClick(action)}
    >
      {action}
    </button>
  );
};
```